docs/book/platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure.md

```diff
@@ -16,7 +16,576 @@ The hard-to-swallow truth is there is no single standard that unifies all authen
+# A representative use-case
```

Comment:
I was not able to link to that specific line (as it didn't change with this PR), but if you look at this page on the preview, the main headline is duplicated at the start. I would remove the (# Connect ZenML to infrastructure) in line 5.